frontend/src/components/auth/Logout.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+const Logout = () => (
```

Comment:
This looks more like header then Logout as it shows informative messages like Log out and Do come again.